README.md

```diff
@@ -1 +1,167 @@
+
+To get a local copy up and running, follow these steps.
+
+- Create a local directory where you can clone the project.
+- Clone the project into your repository
+- Install the dependencies
+- Open the project on a browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and GitHub
+- A code editor
+- Nodejs
+- A browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+- Create a local directory where you can clone the project.
+- Clone the project into your repository
+- Install the dependencies
+- Open the project on a browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Just run npm i to install all dependencies
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+**Leehaney George**
+
+- GitHub: [@githubhandle](https://github.com/leehaney254)
+- Twitter: [@twitterhandle](https://twitter.com/Lee06785586)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/leehaney-george-0a4a51178/)
+
+**Steven Mukami**
+
+- GitHub: [@githubhandle](https://github.com/stevenmukama)
+- Twitter: [@twitterhandle](https://twitter.com/mukama_steven)
+
+**Muhammad Talha Hanif**
+
+- GitHub: [@githubhandle](https://github.com/Talha-Hanif5666)
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Implement nav-bar**
+- [ ] **Implement more pages**
+- [ ] **Add functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/leehaney254/Castapone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for the great resources shared.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Kindly remove these instructions because they are just for your understanding of how to create a MIT License file 😊 So kindly remove these instructions to make your readme file clean 👍